Dockerfiles/alpine/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/runtime:3.1-alpine
+FROM mcr.microsoft.com/dotnet/runtime:5.0-alpine
```

Comment:
@matt-richardson I don't understand this, given we have self-contained packages, should this docker images be using them and hence the image be `mcr.microsoft.com/dotnet/runtime-deps:5.0-alpine`
Sorry I know if not really related, so feel free to ignore

Reply:
I started down the path of switching over to using a more base container, but it got a bit out of scope for what I was trying to do. I think we could (should?) switch to using a smaller base image but that can be tackled separately.